### DISCUSSION TITLE
Etsy shipping profiles limit parameter is not applied

### DISCUSSION THREAD

**Discussion by nikolajevs86** [etsy/open-api#757] [Q&A]
Hi, we need to apply a limit for the shipping profiles as it returns an enormous amount of shipping profiles, which breaks our integration. But seems the limit parameter is not applied, as it was said in the documentation:
https://openapi.etsy.com/v3/application/shops/{shop_id}/shipping-profiles?limit=100 
https://developers.etsy.com/documentation/essentials/urlsyntax/#standard-parameters

Can you please share how we can apply the limit as it breaks our Etsy integrations for a lot of merchants?

**Comment by etsyachristensen** [comment]
@nikolajevs86 This has been queued for review. We've duplicated the issue so we are able to proceed with fixing. We will let you know when the fix is in place.
